ote/src/clj/ote/netex/netex.clj

```diff
@@ -0,0 +1,168 @@
+(ns ote.netex.netex
+  "Scheduled tasks to update gtfs file to s3 and later to database."
+  (:require
+    [taoensso.timbre :as log]
+    [clojure.java.io :as io]
+    [clojure.test :refer [is]]
+    [cheshire.core :as cheshire]
+    [ote.db.netex :as netex]
+    [specql.core :as specql]
+    [clojure.java.shell :refer [sh with-sh-dir]]
+    [clojure.string :as str]
+    [amazonica.aws.s3 :as s3]))
+
+(defn- path-allowed? [^String path]
+  ;; Sanity check to avoid operating in any system or app directory
+  (str/starts-with?
+    (.getAbsolutePath (clojure.java.io/file path))          ; TODO: should use .getCanonicalPath but osx
+    "/tmp/"))
```

Comment:
Move comment to comment
(defn- path-allowed? 
"Sanity check to avoid operating in any system or app directory"
[^String path]

Reply:
Should be comment, will fix